spec/spec_helper_local.rb

```diff
@@ -6,7 +6,7 @@
 RSpec.configure do |config|
   config.mock_with :rspec
```

Comment:
This should be default with voxpupuli-test:
https://github.com/voxpupuli/voxpupuli-test/blob/f6c85fe54412248339cbb8ebe197c0b95dbb5697/lib/voxpupuli/test/spec_helper.rb#L18-L25